From eac9028969471f21c3799c46eb2e24468c019b1c Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Wed, 12 Dec 2012 14:28:47 +0800
Subject: [PATCH 114/310] ENGR00236047 settings: add accounts help url fix
 this issue.

add accounts help url to fix this CR.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 imx6/sabresd/overlay/packages/apps/Settings/res/values/strings.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/imx6/sabresd/overlay/packages/apps/Settings/res/values/strings.xml b/imx6/sabresd/overlay/packages/apps/Settings/res/values/strings.xml
index f240913..00f0cda 100644
--- a/imx6/sabresd/overlay/packages/apps/Settings/res/values/strings.xml
+++ b/imx6/sabresd/overlay/packages/apps/Settings/res/values/strings.xml
@@ -18,4 +18,5 @@
     <!-- Help menu label [CHAR LIMIT=20] -->
     <!-- Help URL, WiFi [DO NOT TRANSLATE] -->
     <string name="help_url_wifi" translatable="false">http://support.google.com/wifi</string>
+    <string name="help_url_accounts" translatable="false">http://support.google.com/accounts</string>
 </resources>
-- 
1.8.0

